source/module_hamilt_lcao/module_gint/gtask_force.cpp

```diff
@@ -211,16 +213,16 @@ void gpu_task_generator_force(const Grid_Technique& gridt,
         }
     }
     atom_pair_num = tid;
-
-    delete[] gpu_mat_cal_flag;
+    gpu_mat_cal_flag.clear();
```

Comment:
you should remove all std::vector.clear() functions

Reply:
It has been resolved